app/src/components/Detail.test.js

```diff
@@ -0,0 +1,19 @@
+import Detail from './Detail'
+
+afterAll(async () => {
+  await cleanup()
+})
+
+describe('Fragments: Detail.js', () => {
+  test('It should render Detail screen', () => {
+    const subscription = {
+      title: 'Spotify'
+    }
+    const handleDelete = () => {}
+    const { container } = render(
+      <Detail handleDelete={handleDelete} subscription={subscription} />
+    )
+
+    expect(container).toBeInTheDocument()
```

Comment:
good job, we can add more expected, if you have a state or value was render (show for user).
expect(assertTarget).toBeVisible()

Reply:
Thanks for review, will merge it 🚀